variants/Generic_F446Rx/variant.h

```diff
@@ -0,0 +1,144 @@
+#define PC5  A15 // | 36      | A15 (ADC1) |            | USART3_RX             |                      |                                   |           |
+#define PC6  37  // | 37      |            |            | USART6_TX             | (FMP_TWI1_SCL)       |                                   |           |
+#define PC7  38  // | 38      |            |            | USART6_RX             | (FMP_TWI1_SDA)       | SPI2_SCK                          |           |
+#define PC8  39  // | 39      |            |            |                       |                      |                                   |           |
+#define PC9  40  // | 40      |            |            | USART3_TX             | TWI3_SDA             |                                   |           |
+#define PC10 41  // | 41      |            |            |                       |                      | SPI3_SCK                          |           |
+#define PC11 42  // | 42      |            |            | USART3_RX, (UART4_RX) |                      | SPI3_MISO                         |           |
+#define PC12 43  // | 43      |            |            | UART5_TX              | TWI2_SDA             | SPI3_MOSI                         |           |
+#define PC13 44  // | 44      |            |            |                       |                      |                                   |           |
+#define PC14 45  // | 45      |            |            |                       |                      |                                   | OSC32_IN  |
+#define PC15 46  // | 46      |            |            |                       |                      |                                   | OSC32_OUT |
+//                  |---------|------------|------------|-----------------------|----------------------|-----------------------------------|-----------|
+#define PD2  47  // | 47      |            |            | UART5_RX              |                      |                                   |           |
+//                  |---------|------------|------------|-----------------------|----------------------|-----------------------------------|-----------|
+#define PH0  48  // | 48      |            |            |                       |                      |                                   | OSC_IN    |
+#define PH1  49  // | 49      |            |            |                       |                      |                                   | OSC_OUT   |
+//                  |---------|------------|------------|-----------------------|----------------------|-----------------------------------|-----------|
+
+/// This must be a literal
+#define NUM_DIGITAL_PINS        50
+#define NUM_ANALOG_INPUTS       16
+
+// On-board LED pin number
+#ifndef LED_BUILTIN
+#define LED_BUILTIN             PA5
+#endif
+#define LED_GREEN               LED_BUILTIN
+
+// On-board user button
+#define USER_BTN                PC13
```

Comment:
This will allow to override this definition
```suggestion
#ifndef USER_BTN                
#define USER_BTN                PC13
#endif
```